README.rst

```diff
@@ -254,6 +254,74 @@ The ``DepositBox`` overrides the record's lifecycle management box on the deposi
 It takes care of rendering the "publish" button only when appropriate in the curation workflow.
 The other ``curationComponentOverrides`` provide better rendering for the new elements (e.g. event types) in the request page.
 
+Set Curation request custom field
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Because there is a need to inform the user about the curation workflow, a custom field at the end is put just to show a specific message. In order to set this up, a basic invenio custom field in your instance should be configured.
```

Comment:
I would relax the language here a bit to highlight that this is optional.
At TUW we've been running Invenio-Curations in production for close to 2 months now, and not added a custom field or banner to hint towards the curation workflow.

The wording on our button seems to be sufficient for our users.

![image](https://github.com/user-attachments/assets/725fefce-abb2-4639-8165-aadfd4940a2a)